inc/classes/class-custom-nav-menu-fields.php

```diff
@@ -361,7 +361,7 @@ public function walker_nav_menu_start_el( $html, $item, $depth, $args ) {
 				if ( ! empty( $data['media-link'] ) ) {
 					$image_html .= sprintf( '<a href="%s">', esc_url( $data['media-link'] ) );
 				}
-				$image_html .= sprintf( '<img class="%s-image" src="%s">', esc_attr( $this->meta_key ), esc_url( $data['media-url'] ) );
+				$image_html .= sprintf( '<img class="%s-image" src="%s">', esc_attr( $this->meta_key ), esc_url( $data['media-link'] ) );
```

Comment:
Can we please use numbered specifiers in `sprintf`
```suggestion
				$image_html .= sprintf( '<img class="%1$s-image" src="%2$s">', esc_attr( $this->meta_key ), esc_url( $data['media-link'] ) );
```

Reply:
on it 👍